app/controllers/api/v1/authors_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V1
+    class AuthorsController < ApplicationController
+      before_action :set_author, only: [:show, :update, :destroy]
+
+      # GET /authors
+      def index
+        authors = Author.all
+        respond_with authors, class: SerializableAuthor
+      end
+
+      # GET /authors/1
+      def show
+        respond_with @author, class: SerializableAuthor, include: [:books]
```

Comment:
since we're ditching the before_action flow, there is no need to use instance variables. just use:

```ruby
respond_with Author.find(params[:id]) ....
```